### DISCUSSION TITLE
백엔드 서버 관리자 상시 모집합니다

### DISCUSSION THREAD

**Discussion by jimin52** [jiphyeonjeon-42/backend#631] [Announcements]
### 지원자격
- Inception 통과자
- 서버 설정과 관리에 관심있는 사람
- 슬랙 알림 항시 켜두는 사람
- 깃허브 알림 켜두는 사람
- 디스커션 상시 주시자

### 하고자 하는 일
- 서버 터지면 고치기
- 서버 보안 상태 체크
- 주말마다 배포하기
- DB 서버 분리하기
- 서버 통계 수집해서 알맞은 서버 구성하기
- 자동배포 최적화하기 (더 빠르고 안정적인 빌드 및 배포)

**Comment by scarf005** [comment]
와! 서버 배포!

**Comment by nyj001012** [comment]
member 되면 해야지 헿

**Comment by Yonaim** [comment]
지원자격...쉽지않군...

**Reply by jimin52** [reply]
인셉션 통과 하셨다면 나머지는 트레이닝 해드립니다!
